.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release Packages
```

Comment:
@thlmenezes could you please fix linter errors before merging this PR?
https://github.com/devopness/devopness/actions/runs/11633880424/job/32399935648?pr=1003#step:6:13 

<img width="880" alt="image" src="https://github.com/user-attachments/assets/0e158324-7c31-406c-9ddf-fddb8ea66ab7">

Reply:
Thanks for the review, was not expecting any lint errors without any code changes on the package files that did not happen before 👀 

Apparently was due to some missing dependencies and configurations.

1. Issues resolved;
1. Updated ESLint configuration to use `import` plugin recommended defaults;
1. Updated lint command to fail on warnings as well, allowing for stricter CI validations.

Could you review this PR again?